core/services/chainlink/application.go

```diff
@@ -238,6 +245,10 @@ func NewApplication(config *orm.Config, ethClient eth.Client, advisoryLocker pos
 		balanceMonitor = &services.NullBalanceMonitor{}
 	}
 
+	subservices = append(subservices, balanceMonitor)
```

Comment:
Do we use the null balanceMonitor if `BALANCE_MONITOR_ENABLED=false`?

Reply:
Yeah, it will still show in the monitoring output, it just always returns ready & healthy